frontend/src/api/billing/manage.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'api';
+import { ErrorResponseHandler } from 'api/ErrorResponseHandler';
+import { AxiosError } from 'axios';
+import { ErrorResponse, SuccessResponse } from 'types/api';
+import {
+	CheckoutRequestPayloadProps,
+	CheckoutSuccessPayloadProps,
+} from 'types/api/billing/checkout';
+
+const manageCreditCardApi = async (
+	props: CheckoutRequestPayloadProps,
+): Promise<SuccessResponse<CheckoutSuccessPayloadProps> | ErrorResponse> => {
+	try {
+		const response = await axios.post('/portal', {
```

Comment:
// Todo

Retrun axios.post directly so that the useMutation call handles onSuccess and onError automatically.